### PR TITLE
Completed string-manipulation-1 exercise.

### PR DESCRIPTION
**Pull request by ryan-byrnes**
Screen Shot 2021-07-02 at 5 35 49 PM

### CODE REVIEW

File: string-manipulation-1/src/get-words.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* exported getWords */
+/* receive string
+iterate through string looking for a space character that indicates the end of a word
+push word into an array
+push final characters into array (final word of the array) */
+
+function getWords(string) {
+  var wordsArray = [];
+  for (var i = 0; i < string.length; i++) {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/get-words.js:9]: This loop isn't necessary

File: string-manipulation-1/src/is-vowel.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported isVowel */
+/* receive string argument
+check whether or not the string character is a vowel
+store boolean result in a variable
+return variable */
+
+function isVowel(char) {
+  return /[aeiou]/.test(char.toLowerCase());
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:8]: You should do this again without regex

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good start here! A couple notes for you:

**Comment by ryan-byrnes** [comment]
Screen Shot 2021-07-06 at 2 48 51 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**